opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -113,10 +113,30 @@ class BatchExportSpanProcessor(SpanProcessor):
     def __init__(
         self,
         span_exporter: SpanExporter,
-        max_queue_size: int = 2048,
-        schedule_delay_millis: float = 5000,
-        max_export_batch_size: int = 512,
+        max_queue_size: int = None,
+        schedule_delay_millis: float = None,
+        max_export_batch_size: int = None,
+        export_timeout_millis: float = None,
     ):
+
+        if max_queue_size is None:
+            max_queue_size = Configuration().get("BSP_MAX_QUEUE_SIZE", 2048)
```

Comment:
It would be good to have environment variables in a common location, we're starting to have a lot of them.

Reply:
I think this is a bit outside of the scope of this PR, opened #1147 to track this.